### PR TITLE
update data jobs monitoring setup guide wrt databricks integration permissions

### PR DESCRIPTION
**Pull request by yiliangzhou**
### What does this PR do? What is the motivation?

This update is to call out some important steps in data jobs monitoring setup wrt the access token permission during Databricks Integration setup:
* the token needs to never expire.
* the token needs to have Can View ACL permissions to Databricks Jobs.
* call out an alternative way of generating access token (as service principal is recommended way of managing identity in Databricks for automated tools)

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/data_jobs/databricks.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 ### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
-1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
+1. Click **Generate new token**, enter a comment, leave the Lifetime as empty and click **Generate**. Take note of your token.
```
Comment (alai97) [inline content/en/data_jobs/databricks.md:26]:
```suggestion
1. Click **Generate new token**, enter a comment, leave the Lifetime as empty, and click **Generate**. Take note of your token.
```

File: content/en/data_jobs/databricks.md [hunk 2]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
...
+1. As an alternative, follow [the guide][10] to generate access token for a [service principal][11].
```
Comment (alai97) [inline content/en/data_jobs/databricks.md:27]:
```suggestion
1. As an alternative, follow the [official Databricks documentation][10] to generate access token for a [service principal][11].
```

File: content/en/data_jobs/databricks.md [hunk 3]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
...
+1. Ensure the account generating the token has [CAN VIEW][9] access for Databricks Jobs you want to monitor and the token is configured to never expire.
```
Comment (alai97) [inline content/en/data_jobs/databricks.md:28]:
```suggestion
1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs you want to monitor, and that the token is configured to never expire.
```

**Review by alai97 — COMMENTED** [review]
Looks good, some suggestions.

**Review by warrierr — APPROVED** [review]
LGTM after you make Alpine's changes

File: content/en/data_jobs/databricks.md [hunk 4]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 ### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
```
Comment (warrierr) [inline content/en/data_jobs/databricks.md:25]:
```suggestion
1. In your Databricks workspace, click on your profile in the top right corner and go to **Settings**. Select **Developer** in the left side bar. Next to **Access tokens**, click **Manage**.
```

File: content/en/data_jobs/databricks.md [hunk 5]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
...
-1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
+1. Click **Generate new token**, enter "Datadog Integration" in the **Comment** field, remove the default value in **Lifetime (days)**, and click **Generate**. Take note of your token. **Important:** Make sure you delete the default value in **Lifetime (days)** so that the token doesn't expire and the integration doesn't break.
```
Comment (warrierr) [inline content/en/data_jobs/databricks.md:26]: Is it possible to get the **Important:...**  callout to be on a newline but not create a separate step in markdown?

File: content/en/data_jobs/databricks.md [hunk 6]
```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
...
+1. As an alternative, follow the [official Databricks documentation][10] to generate access token for a [service principal][11].
+1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs you want to monitor, and that the token is configured to never expire.
```
Comment (warrierr) [inline content/en/data_jobs/databricks.md:28]:
```suggestion
1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs and clusters you want to monitor, and that the token is configured to never expire.
```

**Review by warrierr — CHANGES_REQUESTED** [review]

File: content/en/data_jobs/databricks.md (hunk 6 above)
Comment (warrierr) [reply content/en/data_jobs/databricks.md:28]: Also, the "as an alternative..." and "ensure account generating token ...." shouldn't be explicity steps, but additional details/alternatives on step 2

**Review by alai97 — APPROVED** [review]
LGTM!

**Review by warrierr — APPROVED** [review]

**Review by alpinet — APPROVED** [review]

**Comment by alpinet** [comment]
looks good!

**Pull request merged**